### ISSUE TITLE
SCPH-77004: Unable to dump bios to USB or host

### ISSUE THREAD

**Issue by sestain** [F0bes/biosdrain#20]
Doesn't detect USB and running through PS2Link tries to dump to host but doesn't transfer anything.

PS2Client output:
loadelf: fname host:biosdrain.elf secname all
Input ELF format filename = host:biosdrain.elf
1 00100000 001fe688 ................................
Loaded, host:biosdrain.elf
start address 0x100b70
gp address 00000000
framebuffer at 00000000
zbuffer at 00046000
open name rom0:ROMVER flag 1 data 41378
open fd = 2
biosdrain - revision v2.1.3
mkdir name host:/tmp
mkdir wrong mode, using fallback value 493
rmdir name host:/tmp
loadmodule: fname rom0:ADDDRV args 0 arg
loadmodule: id 31, ret 1
loadmodule: fname rom0:ADDROM2 args 0 arg
loadmodule: id -203, ret 1
SYSMAN: Initializing ROM support.
SYSMAN: EROM filesystem driver not found.
SYSMAN: Initializing i.Link support.
SYSMAN: Initializing DEV9 support.
SYSMAN: Initializing USB support.
SYSMAN: Initializing SPU2 support.
SYSMAN: IOP peripherals initialized successfully.
SYSMAN ID is 32 (res 0)
 Dumping to HOST

**Comment by F0bes** [comment]
Thanks for reporting. I just migrated to a new FAT driver in PR #21 (just waiting for it to be tested).
Unfortunately, I can not get large partitions to work with the new or old drivers.

Try reducing your partition size to ~1GB and leaving the rest unformatted, that is what worked for me.
As for the issues with `host:`, I tested here and it worked fine for me. Very odd how it was able to transfer biosdrain over, but no files back...
